src/VisualizationTests/HelixWatch3DViewModelTests.cs

```diff
@@ -816,6 +816,53 @@ public void Display_Geometry_Labels()
             Assert.AreEqual("[0,0,1]", geometry.TextInfo[0].Text);
         }
 
+        [Test]
+        // This test will select a sphere object 70 times from a list of sphere's, to display 
+        // the corresponding label for that sphere object. After the Helix update, this workflow was causing
+        // delays and would cause dynamo to hang. The fix was added in this PR: https://github.com/DynamoDS/Dynamo/pull/10399
+        // Before the fix, this test would take around 5 mins to finish but now this test finishes in just 20 secs. 
+        public void PerformanceTestOnLabelsAfterHelixUpgrade()
```

Comment:
do you think it is worth timing this test and asserting it takes less than a minute or something like that?

Reply:
Also, might we pick a lower number than 70 for the test? Since the time execution reduced greatly after the improvement, we could still see a difference with maybe only 7 selections. By doing that we'd reduce the execution time of this test down to a few seconds.

Reply:
For number of selections less than 10, there is not much difference in the test time before and after the fix. For 30 selections, we can see a time difference of around 30-40 secs. As more and more labels are rendered, number of exceptions caught increases and dynamo's performance decreases exponentially. So, I went with a bigger selection number. 
I think 30 should be fine, let me know your thoughts.

Reply:
Sounds good 👍  Thanks @reddyashish

Reply:
Nit: The comments still say 70 after the change

Reply:
Thanks @mmisol . Updated the comments.